src/components/Input.js

```diff
@@ -3,23 +3,22 @@
 import React from 'react';
 import styled from 'styled-components/native';
 
-const InputWrapper = styled.View`
-  padding: 10px;
+const InputWrapper = styled.View`  
   border-bottom-color: ${props => props.theme.colors.secondaryColor};
   border-bottom-width: 2.5;
-  margin-top: 24px;
 `;
 
 const RegisterTextInput = styled.TextInput.attrs({
   placeholderTextColor: props => props.theme.colors.secondaryColor,
-  underlineColorAndroid: props => props.theme.colors.secondaryColor,
+  underlineColorAndroid: 'rgba(0,0,0,0)',
```

Comment:
use props.theme.colors

Reply:
@jaburcodes this is the transparent color this dont need to use theme provider